.golangci.yaml

```diff
@@ -106,6 +95,61 @@ linters-settings:
   stylecheck:
     checks: ["ST1019"]
 
+  sloglint:
+    # Enforce not mixing key-value pairs and attributes.
+    # https://github.com/go-simpler/sloglint?tab=readme-ov-file#no-mixed-arguments
+    # Default: true
+    no-mixed-args: true
+    # Enforce using key-value pairs only (overrides no-mixed-args, incompatible with attr-only).
+    # https://github.com/go-simpler/sloglint?tab=readme-ov-file#key-value-pairs-only
+    # Default: false
+    kv-only: true
```

Comment:
nit: Why? I understand `no-mixed-args` but what's the reason to avoid Attrs?

Reply:
From a quick benchmark it showed that Attrs are worse in terms of performance, so to be consistent I figured it would be better to only allow kv instead of allowing mixed-args.

Reply:
I'm a bit torn, [slog docs](https://pkg.go.dev/golang.org/x/exp/slog#hdr-Attrs_and_Values) claim that attrs should be the fastest way to log, as the type information doesn't have to inferred. With this, we're forcing devs to not do what the docs say.

I guess it'd be nice to have a clear signal to revert this when the underlying slog implementation changes behaviour to be more consistent with the docs, but it's mostly an academic concern.